V_HexMetric.cpp

```diff
@@ -23,19 +23,18 @@
 #include "VerdictVector.hpp"
 #include "verdict.h"
 
-#include <algorithm>
-#include <cmath> // for std::isnan
+#include <math.h>
 
 namespace VERDICT_NAMESPACE
 {
-extern void quad_minimum_maximum_angle(double min_max_angles[2], const double coordinates[][3]);
+VERDICT_HOST_DEVICE extern void quad_minimum_maximum_angle(double min_max_angles[2], const double coordinates[][3]);
 
-static const double one_third = 1.0 / 3.0;
-static const double two_thirds = 2.0 / 3.0;
-static const double sqrt3 = std::sqrt(3.0);
```

Comment:
Would you prefer if I did something like the following so that sqrt3 is a constant on the host rather than always recomputed?

```
VERDICT_HOST_DEVICE static double sqrt3()
{
#if defined(VERDICT_DEVICE_COMPILE)
  return sqrt(3);
#else
  static const double s_sqrt3 = sqrt(3);
  return s_sqrt3;
#endif
```